LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2021 Chris Brody, Tim Brust, Eric Newport and other contributors
```

Comment:
- I would probably leave the year out, like this one: https://github.com/facebook/react/blob/master/LICENSE
- You forgot the original author @jindw
- I think you should add yourself first if you are planning to make more changes

Reply:
Thank you for pointing to the part that I was least sure about.

> * I would probably leave the year out, like this one: https://github.com/facebook/react/blob/master/LICENSE

According to all templates I found the year is part of them (and it looks to me as if Facebook just took the liberty to drop it).
But we could say `2014 to present` to make it more maintainable.

> * You forgot the original author @jindw
> * I think you should add yourself first if you are planning to make more changes

Should we really keep a redundant list in package.json and this file?
Maybe we can rather link to the package.json file and only mention the current maintainers here?
Or list specify the years that the original author contributed?
Ideally we could mention an entity here, so it doesn't need to be updated so often, but I'm not sure xmldom org would be good enough since it's not a legal entity as far as I know.
If that's not possible I would argue for only mentioning a small amount of people (e.g. the people able to publish the NPM package) and point to the other contributors in some way.

Reply:
I think this answer can help us solve the problem with the redundant list: https://opensource.stackexchange.com/questions/4960/can-a-team-be-the-copyright-holder-mit/4970#4970

In terms of year range, I would make it "2012-present". I think it goes back to 2012 from the original author: https://github.com/xmldom/xmldom/commits/master?after=ad773c90f2e6c080ea552487b7604e9339c2a689+279&branch=master

I am also assuming that it is OK for us to "stamp" it back to 2012 even though we (you and I) did not start contributing until 2019/2020.

Reply:
> I think you should add yourself first if you are planning to make more changes

Oh and I just wanted to support progress on this topic, I'm not eager right now to take more responsibility regarding xmldom than I'm already doing.